bg1npc/bg1npc.tp2

```diff
@@ -34,7 +34,8 @@ LANGUAGE ~Polski (Tlumaczenie przez Children of Bhaal)~ ~polish~ ~bg1npc/tra/pol
 LANGUAGE ~Deutsch (Teiluebersetzung vom Kerzenburgforum, teilweise noch in englisch)~ ~german~ ~bg1npc/tra/german/setup.tra~ // https://www.baldurs-gate.de/index.php
 LANGUAGE ~Russian (by Aerie Team & Arcanecoast Team)~ ~russian~ ~bg1npc/tra/english/setup.tra~ 
 LANGUAGE ~Italian (by Coland)~ ~italian~ ~bg1npc/tra/italian/setup.tra~ 
-LANGUAGE ~Chinese~ ~schinese~ ~bg1npc/tra/schinese/setup.tra~
+LANGUAGE ~Simplified Chinese~ ~schinese~ ~bg1npc/tra/english/setup.tra~
+                                         ~bg1npc/tra/schinese/install.tra~

```

Comment:
@MephistoSatanDevil Please rename the new install.tra file to setup.tra, as the original setup file contains only commands prompt too, then it is the exactly what you need. Simply keep it in UTF-8 and add it to the exclude list.

Reply:
When we simply rename the install.tra file back to setup.tra and keep it in UTF-8 encoding, an error occurs during installation. 
Could you please advise on how to add it to the charset convertion exclude list?

Reply:
That's strange that you have some error during installation. Let me try this and I will come back to you. However I meant exclude list as the variable `x#noconvert` that you've already set in this change:

Reply:
I opened a PR in your repository fork. I was able to install the mod successfully. If you still see the error, please post a error message or screenshots.

Reply:
> I opened a PR in your repository fork. I was able to install the mod successfully. If you still see the error, please post a error message or screenshots.
I see now. 
Because I was afraid it would affect other languages, so  I did not modify the variable x#reload and x#noconvert before. 

So, by making this change, do the setup.tra files for other languages need to be converted to UTF-8 encoding?

Reply:
> Because I was afraid it would affect other languages, so I did not modify the variable x#reload and x#noconvert before.
> So, by making this change, do the setup.tra files for other languages need to be converted to UTF-8 encoding?

This is correct. It is affecting the other languages as well. Still it is better to update those files instead of duplicating the solution and have the same in two files (install and setup) with the same purpose. I will update required files for other languages.

Reply:
> This is correct. It is affecting the other languages as well. Still it is better to update 

those files instead of duplicating the solution and have the same in two files (install and setup) with the same purpose. I will update required files for other languages.

Thank you.

Reply:
PR is ready to be merged in your repo fork.